dracut/30ignition/module-setup.sh

```diff
@@ -62,6 +62,9 @@ install() {
     install_ignition_unit coreos-populate-var.service
     inst_script "$moddir/coreos-populate-var.sh" \
         "/usr/sbin/coreos-populate-var"
+
+    # needed for openstack
```

Comment:
maybe add specifically for config drive support

Reply:
Done.